src/main/java/com/sugarcrm/candybean/automation/element/Element.java

```diff
@@ -53,9 +53,39 @@ public Element(Hook hook, int index) {
 	 */
 	public abstract Element getElement(Hook hook, int index) throws CandybeanException;
 
+	/**
+	 * Executes a javascript command against this element.
+	 * @param	javascript	The javascript code to execute
+	 * @param	javascript	The arguments to pass.  Note that indices of the arguments passed will

```

Comment:
the param key is not javascript in this line, I'm guessing it was supposed to be "args"